### DISCUSSION TITLE
Legal terms of use

### DISCUSSION THREAD

**Discussion by JenPaul79** [rsms/inter#694] [General]
Hi Rasmus, 
I work for a company which is interested to use Inter font for its websites and App, so we are talking about of a Web License including a commercial scope. I've read your license.txt and basing on it seems that the above uses it's free being under Open Source licenze (SIL). 
May you confirm me that?

**Comment by rsms** [comment]
Hello @JenPaul79. What do you want me to confirm? I don't understand your question. Are you asking if you can use Inter in a product with commercial purposes?

The license (https://github.com/rsms/inter?tab=OFL-1.1-1-ov-file#readme) makes the conditions very clear. There's even an entire website dedicated to helping you understand the OFL license: https://openfontlicense.org/how-to-use-ofl-fonts/

**Comment by JenPaul79** [comment]
Hello, 
The question was if we could use freely Inter font to create text contents on our website and app, which have a commercial scope. 
We just asked confirmation on that.

Thank you so much!

**Reply by rsms** [reply]
Yes you can

**Reply by JenPaul79** [reply]
Thank you so much!
